src/background/services/mnemonic/index.ts

```diff
@@ -0,0 +1,8 @@
+import * as bip39 from "bip39";
+
+export const generateMnemonic = (): string => bip39.generateMnemonic();
+
+export const validateMnemonic = (mnemonic: string): boolean => bip39.validateMnemonic(mnemonic);
+
+export const mnemonicToSeed = (mnemonic: string): Promise<string> =>
+  bip39.mnemonicToSeed(mnemonic).then((bytes) => bytes.toString("hex"));
```

Comment:
Do you think we should also enforce using {} braces here as well in functions as we done in conditions?

Reply:
For arrow functions no. If there is more than 1 line we can use `function`.